<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ian Murray</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-development">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Game List</string>
   </property>
   <widget class="QPushButton" name="cloneButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Games &amp;&amp; Clones from CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Controls from JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mappingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Mappings to Game List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xmlButton">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load mame.xml File</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="mergeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Merge Data Files</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>xmlButton</tabstop>
  <tabstop>cloneButton</tabstop>
  <tabstop>controlButton</tabstop>
  <tabstop>mappingButton</tabstop>
  <tabstop>mergeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
